docs/conversion_examples_gallery/recording/intan.rst

```diff
@@ -27,4 +33,144 @@ Convert Intan data to NWB using :py:class:`~neuroconv.datainterfaces.ecephys.int
     >>> metadata["NWBFile"].update(session_start_time=session_start_time)
     >>>
     >>> nwbfile_path = f"{path_to_save_nwbfile}"  # This should be something like: "./saved_file.nwb"
-    >>> interface.run_conversion(nwbfile_path=nwbfile_path, metadata=metadata)
+    >>> interface.run_conversion(nwbfile_path=nwbfile_path, metadata=metadata, overwrite=True)
+
+Intan Analog Data Conversion
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Convert non-amplifier analog data from Intan systems to NWB using :py:class:`~neuroconv.datainterfaces.ecephys.intan.intananaloginterface.IntanAnalogInterface`.
```

Comment:
The `IntanAnalogInterface` link in the docs is not clickable, maybe it will be after merge?

<img width="390" height="323" alt="image" src="https://github.com/user-attachments/assets/9f52145f-c118-4b76-b53d-b72651bc95b4" />

Reply:
Thanks for pointing that out. I will fix it.

Reply:
I forgot to add an entry on the API. It is clickable now.

Reply:
Thanks!